.teamcity/src/main/kotlin/configurations/CompileAllBuildCacheNG.kt

```diff
@@ -8,10 +10,17 @@ class CompileAllBuildCacheNG(model: CIBuildModel, stage: Stage, oldCacheWithNgRe
         id("${model.projectId}_CompileAllBuild_NGRemote")
         name = "Compile All With NG Remote"
         description = "Compiles all production/test source code using old build cache but NG remote"
+
+        vcs {
+            checkoutDir = "ng-remote-${DslContext.uuidPrefix}"
+        }
```

Comment:
🤔 If I understand correctly, this doesn't need its own checkout dir, since it is using the old build cache that doesn't have the problems.

Reply:
It's using old cache, but the same node as new cache. So if we want to be 100% on a safe side, then this makes sense. 

But if it's expensive, then we also don't need it. I kinda trust that our implementation that old cache won't pull new cache entries.